.gitignore

```diff
@@ -16,3 +16,6 @@ vendor/*
 
 .kitchen/
 .kitchen.local.yml
+
+**/stderr.txt
```

Comment:
Where do these files come from?

Reply:
They are generated during the unit tests to remove any unnecessary output generated during the tests from being logged to the console while still capturing it for debugging purposes.